packages/library/README.md

```diff
@@ -34,17 +34,17 @@ In response to your feedback, we began a process of modernizing the library to p
 ### For use in Node.js or a web application
 
 ```shell
-npm install --save @solana/web3.js@tp
+npm install --save @solana/web3.js@preview
```

Comment:
Which of these are we doing?

* Tag this thing `tp2` and have the preview line continue on.
* Tag this thing `preview` and have people who install Technology Preview 2 get automatically upgraded to Technology Preview X when we make more?

I sort of don't know when this line is going to end. Maybe the next thing we release is Release Candidate, tbf.

Reply:
> Tag this thing `tp2` and have the preview line continue on.

I would say this makes more sense.

> Tag this thing `preview` and have people who install Technology Preview 2 get automatically upgraded to Technology Preview X when we make more?

I wouldn't automatically update anyone, since we aren't really stable yet and things could change.

Reply:
Let's use `tp2` then. 👍